core/tests/protractor_desktop/topicAndStoryEditor.js

```diff
@@ -139,6 +139,64 @@ describe('Topic editor functionality', function() {
     });
   });
 
+  it(
+    'should edit the contents of the created question', function() {
+      topicEditorPage.get(topicId);
+      topicEditorPage.moveToQuestionsTab();
+      topicEditorPage.expectNumberOfQuestionsForSkillWithDescriptionToBe(
+        1, 'Skill 1');
+      // Get the questions list and edit the contents of the question.
+      element.all(
+        by.css('.protractor-test-question-list-item')).then(
+        function(questionItems) {
+          questionItems[0].click();
```

Comment:
To clarify, I think this use of indexing is okay because these questions are in a topic "owned" by this test, so there shouldn't be any clashes with other tests